scripts/screeps_rl_experiment_card.py

```
@@ -1076,6 +1174,13 @@ def positive_int_arg(raw: str) -> int:
     return value
 
 
+def loop_a_local_fallback_value(value: int | None, *, default: int, maximum: int, label: str) -> int:
+    resolved = default if value is None else value
+    if resolved > maximum:
+        raise CardValidationError(f"Loop A local fallback {label} must be <= {maximum}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)